docs/products/kafka/kafka-connect/howto/enable-connect.rst

```diff
@@ -0,0 +1,17 @@
+Enable Kafka Connect on Aiven for Apache Kafka
+==============================================
+
+Kafka Connect can be enabled on the same nodes running Aiven for Apache Kafka with **business** or **premium** plans. 
```

Comment:
Can it be enabled on different nodes? I wonder if we rephrase 

To enable  Kafka Connect for Aiven for Apache Kafka your services should be running on **business** or **premium** plans.

Do we loose some meaning?

Reply:
Welll, we need to say that with this method we are running Kafka connect on the same nodes where Kafka is running, you could always create a dedicated cluster. I'll rephrase